### PR TITLE
pl-2250: add ability to set language

### PR DESCRIPTION
**Pull request by nikitarui92**
We need to set `language` for sdk client

### CODE REVIEW

File: template/requester.ts [hunk 1]
```diff
@@ -168,6 +173,9 @@ export class SdkRequester {
     if (idempotencyKey) {
       headers['Idempotency-Key'] = idempotencyKey;
     }
+    if (this.locale) {
+      headers['X-Language'] = this.locale;
```
Comment (dmlyapin) [inline template/requester.ts:177]: use lowercase, like for `'x-access-token`

**Review by dmlyapin — APPROVED** [review]

File: template/requester.ts (hunk 1 above)
Comment (nikitarui92) [reply template/requester.ts:177]: http header is case-insensetive
why we need lowercase here?

Comment (dmlyapin) [reply template/requester.ts:177]: to make it same with `x-access-token`

Comment (nikitarui92) [reply template/requester.ts]: done

**Pull request merged**